content/en/docs/aws/customizing-aws.md

```diff
@@ -78,7 +78,7 @@ spec:
     username: admin@kubeflow.org
 ```
 
-If you use {{% config-uri-aws-cognito %}}, please see [this section](/docs/aws/authentication)
+If you use [configuration file with cognito]({{% config-uri-aws-cognito %}}), please see [this section](/docs/aws/authentication)
 
 ### Customize IAM Role for Pods
 Please see [this section](/docs/aws/iam-for-sa)
```

Comment:
```suggestion

Refer to the [Setting up AWS IAM role for Service Account](/docs/aws/iam-for-sa) guide.
```

("see" -> "refer to" or "go to" for accessibility)
(use full name of guides for better UX)